ms_logs/logs.txt

```diff
@@ -0,0 +1,8 @@
+Mon Feb 21 02:24:44 UTC 2022 | User signed up:Alok | USERNAME:N/A
```

Comment:
If we are writing to the server, why do we have a log file here?

Reply:
Removed